mopidy/ext.py

```diff
@@ -178,27 +188,27 @@ class Registry(Mapping):
     e.g. ``local:foo`` or ``http:bar``.
     """
 
-    def __init__(self):
-        self._registry = {}
+    def __init__(self) -> None:
+        self._registry: Dict[str, List[Type[Any]]] = {}
 
-    def add(self, name, cls):
+    def add(self, name: str, cls: Type[Any]) -> None:
         """Add a component to the registry.
 
         Multiple classes can be registered to the same name.
         """
         self._registry.setdefault(name, []).append(cls)
 
-    def __getitem__(self, name):
+    def __getitem__(self, name: str) -> List[Type[Any]]:
         return self._registry.setdefault(name, [])
 
-    def __iter__(self):
+    def __iter__(self) -> Iterator[str]:
         return iter(self._registry)
 
-    def __len__(self):
+    def __len__(self) -> int:
         return len(self._registry)
 
 
-def load_extensions():
+def load_extensions() -> List[ExtensionData]:
```

Comment:
Should 
```
installed_extensions = []
```
on line 217 also get this type?

Reply:
I believe that is inferred from that variable being used as the return value?

Reply:
OK, makes sense.